index.js

```diff
@@ -205,7 +211,11 @@ var TruffleContractSchema = {
     // update schema version
     normalized.schemaVersion = pkgVersion;
 
-    return normalized;
+    if (options.validate) {
```

Comment:
When you say "link references", so you mean the `linkReferences` key? If not, in general, the way we perform links now *should* go in the schema - that is, network objects's `links` key should be validated to have the right structure.

Remember: The new schema is the authority on what is and isn't correct, so if the tests are sending in bad links we should change the tests (on `develop`, of course).